Marlin/src/HAL/HAL_STM32/pinsDebug_STM32GENERIC.h

```diff
@@ -29,9 +29,11 @@
 #ifdef BOARD_NR_GPIO_PINS  // Only in STM32GENERIC (Maple)
 
 #ifdef __STM32F1__
-  #include "../HAL_STM32F1/fastio_STM32F1.h"
+  #include "../HAL_STM32F103R/fastio.h"
```

Comment:
no no no :p F1 HAL is ok for all of them

Reply:
But if I renamed the folder and the fastio file then it is like no change at all.

Reply:
just make no sense to change that, and mayb bug all the forks

Reply:
Did you not suggest to rename `HAL_STM32F1` to `HAL_STM32F103R` earlier?

> "the F1 is only for the 103R devices... maybe it should be renamed to F103R"

Reply:
> bug all the forks

Rename of folders and files is included in `git merge`.

Reply:
i meant the F1 "env" section

Reply:
in the same register, the F7 env is using a card not even in the st framework

Reply:
> the F7 env is using a card not even in the st framework

I'll have to leave cleanup of these areas to the STM experts.

So… I'll just undo that HAL rename now! 😜

Reply:
> i meant the F1 "env" section

Great. And later we can add more STM32F1… envs for the other variants as they come along.

Reply:
see PR #14846